### PR TITLE
Minor fixes/improvements in Metric docs

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Includes minor fixes in the Metric docs which I realised while reading.

- Include import statement in all examples.
- Fix some rendering issues.
- [**RFC** on 6447e32] Change paths to class names, e.g. `pl.metrics.classification.Accuracy` -> `pl.metrics.Accuracy`.
  *Before*
  
  *After*
  

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/metrics/classification/precision_recall_curve.py [hunk 1]
```diff
@@ -131,7 +133,8 @@ def compute(
         """
         Compute the precision-recall curve
 
-        Returns: 3-element tuple containing
+        Returns:
+            3-element tuple containing
```
Comment (akihironitta) [inline pytorch_lightning/metrics/classification/precision_recall_curve.py:137]: This fixes weird rendering.

File: pytorch_lightning/metrics/functional/precision_recall.py [hunk 2]
```diff
@@ -81,9 +81,6 @@ def precision(
             Note that what is considered a sample in the multi-dimensional multi-class case
             depends on the value of ``mdmc_average``.
 
-        class_reduction:
-            .. warning :: This parameter is deprecated, use ``average``. Will be removed in v1.4.0.
-
```
Comment (akihironitta) [inline pytorch_lightning/metrics/functional/precision_recall.py:86]: duplicate

**Review by Borda — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

File: pytorch_lightning/metrics/classification/auroc.py [hunk 3]
```diff
@@ -39,34 +39,35 @@ class AUROC(Metric):
...
+           before returning the value at the step.
+       process_group:
+           Specify the process group on which synchronization is called. default: None (which selects the entire world)
+       dist_sync_fn:
+           Callback that performs the allgather operation on the metric state. When ``None``, DDP
+           will be used to perform the allgather
 
     Example (binary case):
 
+        >>> from pytorch_lightning.metrics import AUROC
```
Comment (rohitgr7) [inline pytorch_lightning/metrics/classification/auroc.py:70]: are these required? considering that the example is under the section of it's respective class/function

Comment (akihironitta) [reply pytorch_lightning/metrics/classification/auroc.py:70]: Well, they're completely optional, but I, as a user, find it useful to have import statements in examples. I can revert it and delete all import statements right below classes/functions from the docs if agreed.

Comment (rohitgr7) [reply pytorch_lightning/metrics/classification/auroc.py:70]: cool :+1:, let's keep them. sklearn have it too.

**Review by tchaton — APPROVED** [review]
Great PR !

**Review by justusschock — APPROVED** [review]

**Comment by ydcjeff** [comment]
Seems like it is still on master : https://pytorch-lightning.readthedocs.io/en/latest/extensions/metrics.html#accuracy
